packages/block-library/src/query/edit/index.js

```diff
@@ -19,12 +21,12 @@ import QueryInspectorControls from './query-inspector-controls';
 import { DEFAULTS_POSTS_PER_PAGE } from '../constants';
 
 const TEMPLATE = [ [ 'core/query-loop' ] ];
-export default function QueryEdit( {
+export function QueryEditContainer( {
```

Comment:
Wouldn't this make more sense as `QueryContent`?

Reply:
Yes it would. I just made the draft PR to look how it feels and will polish it now.